lib/matint.gi

```diff
@@ -237,7 +237,7 @@ local k,g,b,ii,m1,m2,t,tt,si,n,m,i,j,r,jj,piv,d,gt,tmp,A,T,TT,kk;
 end);
 
 BindGlobal("BITLISTS_NFIM",
-  [ [ false, false, false, false, false ], [ true, false, false, false, false ],
+MakeImmutable( [ [ false, false, false, false, false ], [ true, false, false, false, false ],
     [ false, true, false, false, false ], [ true, true, false, false, false ],
```

Comment:
I agree. Now you've mentioned this, I'm wondering if I should go and backtick HPC-GAP, and then re-make this commit, or just do the merge and then backtick later (that's my temptation to be honest, mainly to avoid re-doing this merge)